onnxruntime/python/tools/kernel_explorer/kernels/skip_layer_norm_test.py

```diff
@@ -14,7 +14,7 @@
 def get_bert_sizes():
     batch_sizes = [1, 8, 64, 128]
     seq_lens = [64, 128, 256, 384, 512]
-    hidden_sizes = [768, 1024]
+    hidden_sizes = [768, 1024, 2048]
     return product(batch_sizes, seq_lens, hidden_sizes)
 
```

Comment:
I think we will need more extensive testing here, for example for vairous ld sizes 1, 2, 3, 7, 9, 13, 63, 65, 127, 129, 177. The correctness test cases can be a separate set from perf profiling cases.